context_response.go

```diff
@@ -89,6 +89,10 @@ func (r *ContextResponse) Status(code int) contractshttp.ResponseStatus {
 	return NewStatus(r.instance, code)
 }
 
+func (r *ContextResponse) Stream(step func(w contractshttp.StreamWriter) error, code int, contentType string) contractshttp.Response {
```

Comment:
Do we need to set  `contentType` for stream? If the `contentType` is only used by SSE, can we add another `SSEvent` method to deal with it?

Reply:
It is not necessary; we can remove `contentType`. We don't need the `SSEvent` method since we provide the `Header` method, allowing the user to set it directly if needed.
```go
ctx.Response().Header("Content-Type", contentType).Stream(...)
```

Reply:
Great 👍